apps/users/schemas.py

```diff
@@ -221,6 +221,19 @@ class CreateStudentOut(BaseOutSchema):
     student_id: int
 
 
+class CreateStudentsOut(BaseOutSchema):
+    student_id: int
+    first_name: str
```

Comment:
it's better to inherit current class from class FullNameSchema to avoid code duplication
and also... is't possible to re-using schema class StudentsListOut instead of creating new one CreateStudentsOut? bcs as i can see, columns are the same...

Reply:
Can't perform output without newly created CreateStudentsOut, and there is no ability to use FullNameSchema

Reply:
got it, ok